tests/server/disco/TestViews.js

```diff
@@ -51,11 +54,14 @@ describe('Discovery Pane GET requests', () => {
     .then((res) => {
       assert.equal(res.header.location,
         '/en-US/firefox/discovery/pane/48.0/Darwin/normal');
+      return true;
     }));
 
   it('should set an HSTS header', () => request(app)
```

Comment:
This is using implicit returns - is the extra return value still needed in the then if it was an explicit return? 

This is a separate problem but the more I think about it we should look to standardise on explicit returns and never use the the implicit arrow return style.